partitioner/src/main/java/io/confluent/connect/storage/partitioner/TimeBasedPartitioner.java

```diff
@@ -38,16 +40,16 @@
 import java.util.Map;
 
 import io.confluent.connect.storage.common.SchemaGenerator;
-import io.confluent.connect.storage.common.StorageCommonConfig;
+import io.confluent.connect.storage.common.util.StringUtils;
 import io.confluent.connect.storage.errors.PartitionException;
 
 public class TimeBasedPartitioner<T> extends DefaultPartitioner<T> {
-  // Duration of a partition in milliseconds.
   private static final Logger log = LoggerFactory.getLogger(TimeBasedPartitioner.class);
 
   private static final String SCHEMA_GENERATOR_CLASS =
       "io.confluent.connect.storage.hive.schema.TimeBasedSchemaGenerator";
```

Comment:
Does your unit test work? Looks like you don't have access to this class.

Reply:
@johnnycaol Not sure what you mean. `mvn clean test` worked on my machine, yes. And Jenkins says this PR is good. 

This property was added in #31